framework/src/main/java/org/tron/core/net/peer/PeerManager.java

```diff
@@ -94,7 +94,11 @@ private static void remove(PeerConnection peerConnection) {
   }
 
   public static synchronized void sortPeers() {
-    peers.sort(Comparator.comparingDouble(c -> c.getChannel().getAvgLatency()));
```

Comment:
why not use Comparator.comparingLong ? the type of c.getChannel().getAvgLatency() is long

Reply:
Modified in this PR https://github.com/tronprotocol/java-tron/pull/5879